application/main.c

```diff
@@ -1,4 +1,37 @@
+/* Copyright Ville Hiltunen 2018 <hiltunenvillej@gmail.com>
+ *
+ * All code here is open source
+ */
+#include <stdio.h>
+#include <string.h>
 
-int main(int ac, char **av) {
+#include "thread.h"
+#include "leds.h"
+#include "color.h"
+#include "xtimer.h"
+
+
+int main(int ac, char **av)
+{
+    //Initialize LED API. This starts a new thread.
+    leds_init();
+    LED2_TOGGLE;
+    color_rgba_t led_color = {
+        .color = { .r = 255, .g = 0, .b = 0},
+        .alpha = 100
```

Comment:
Add a trailing comma.  While they are optional, they make it easier to modify the code later.